wiki/python/ESTIMATOR_GUIDE.md

```diff
@@ -168,6 +199,17 @@ def get_param_names(self):
 
 **Note:** Be sure to include `super().get_param_names()` in the returned list to properly set the `super()` attributes.
 
+### Estimator Tags and cuML Specific Tags
+
+Scikit-learn introduced estimator tags in version 0.21, which are used to programmatically inspect capabilities of estimars. These includes items like sparse matrix support, need for positive inputs, among multiple other things. cuML estimators support _all_ of the tags defined by the Scikit-learn estimator developer guide, and will add support for any tag added there.
+
+Additionaly, some tags specific to cuML have been added. These tags allow for optimization of data generation besides testing (for example pipelines and HPO), and/or are specific to GPU data types. These are:
```

Comment:
```suggestion
Additionaly, some tags specific to cuML have been added. These tags may or may not be specific to GPU data types and can even apply outside of automated testing, such as allowing for the optimization of data generation. This can be useful for pipelines and HPO, among other things. These are:
```